superset-frontend/src/components/Chart/Chart.jsx

```diff
@@ -299,22 +314,24 @@ class Chart extends React.PureComponent {
         onError={this.handleRenderContainerFailure}
         showMessage={false}
       >
-        <Styles
-          data-ui-anchor="chart"
-          className="chart-container"
-          data-test="chart-container"
-          height={height}
-          width={width}
-        >
-          <div className="slice_container" data-test="slice-container">
-            <ChartRenderer
-              {...this.props}
-              source={this.props.dashboardId ? 'dashboard' : 'explore'}
-              data-test={this.props.vizType}
-            />
-          </div>
-          {isLoading && !isDeactivatedViz && <Loading />}
-        </Styles>
+        <InView triggerOnce onChange={this.handleChartInView}>
```

Comment:
Can we add a [`rootMargin`](https://github.com/researchgate/react-intersection-observer#options) to page bottom so charts near the bottom but below the fold can load faster (so users can see less loading states)?